test/common.js

```diff
@@ -1,7 +1,10 @@
-
-// whatever, it's just tests.
-;["util","assert"].forEach(function (thing) {
-  thing = require("thing")
-  for (var i in thing) global[i] = thing[i]
-}
-
+exports.rmFrom = function rmFrom (obj) {
```

Comment:
While I think `rmFrom` is fine when the function is declared near its context (as it was previously), as a globally available tool I'd like the function name to be more descriptive. `rmFromInShrinkwrap` or something like that?